app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -4,8 +4,10 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Customer(var name: String, var phone: String, @SerializedName("id") var customerId:Int = 0 ) {
-    override fun toString(): String {
-        return name
-    }
+data class Customer(
+    var name: String,
+    var phone: String,
+    @SerializedName("id") var customerId:Int = 0)
+{
+    override fun toString(): String { return name }
```

Comment:
Changes to formatting and breaks in classes to more accurately reflect kotlin conventions